### PR TITLE
Add some notes on the design process

### PR DESCRIPTION
**Pull request by nrc**
r? @rylev Covers our discussion yesterday and some things from Zulip.

### CODE REVIEW

File: surveys/notes/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Notes
```
Comment (rylev) [inline surveys/notes/README.md:1]: Should this whole section be called `docs` or `process` instead?

File: surveys/notes/process.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+See also [design.md](design.md).
+
+We intend to start the survey design process by writing a design document. Starting with the 'requirements': the lessons we hope to learn from the survey, we will work backwards to the data we hope to collect (in broad terms and including the cohorts we may wish to analyze), and finally to the questions we wish to ask in the survey.
+
+
+## Requirement collection
+
+To ensure we meet the requirements of 'clients' of the survey (i.e., people who will consume the processed result data), we intend to ask the following groups of people for input:
+
+TODO tracking issue?
```
Comment (rylev) [inline surveys/notes/process.md:26]: I can create this issue.

File: surveys/notes/process.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Process
+
+Proposed timeline:
```
Comment (rylev) [inline surveys/notes/process.md:3]: This timeline is already included here: https://github.com/rust-lang/surveys/tree/main/surveys/2021-annual-survey#tentative-timeline

Not sure we need both.

**Review by rylev — APPROVED** [review]
Looking good! I had one question about the inclusion of the timeline, but there's no need to block merging this.

File: surveys/notes/README.md (hunk 1 above)
Comment (nrc) [reply surveys/notes/README.md:1]: I thought we might make notes on other things too, so have kept process to one file. I thought 'notes' better reflects the quality of the documentation than 'docs' :-)

**Pull request merged**